dockerfiles/centos_7_1.x

```diff
@@ -60,7 +58,6 @@ RUN set -x \
         lz4-devel \
         binutils-devel \
         ncurses-devel \
-        lua-devel \
```

Comment:
Why we should remove it? Commit message doesn't contain any explanation.

Reply:
lua-devel was used for luarocks build, which is removed for now because of tarantoolctl used.